src/Tribe/Main.php

```diff
@@ -251,6 +251,23 @@ public function load_assets() {
 			]
 		);
 
+		tribe_asset(
```

Comment:
We should update this to stellarwp/assets

```php
Asset::add( 'tec-copy-to-clipboard', 'utils/tec-copy-to-clipboard.js' )
	->set_dependencies( 'tribe-clipboard' )
	->enqueue_on( 'admin_enqueue_scripts' )
	->add_localize_script(
		'tribeCopyToClipboard',
		[
			'name' => 'tribeCopyToClipboard',
			'data' => [
				'clipboard_copied_text' => _x( 'Copied to Clipboard!', 'Copy to clipboard success message', 'tribe-common' ),
				'clipboard_fail_text'   => _x( 'Failed to copy.', 'Copy to clipboard failed message', 'tribe-common' ),
			],
		]
	)
	->register();
```

Reply:
@borkweb thank you very much for your suggestions! I still think its a positive think we have a wrapper in our codebase instead of accessing Stellar Assets directly. It is possible in the future we would need to change again, only changing our wrapper is very convenient i believe. Let me know if you still think i should go ahead with this change.

Reply:
Great question!

Stellar assets was built specifically to replace tribe_asset(). While that function was set to use Stellar Assets behind the scenes, that was to allow us to shift without needing to change code everywhere. 

So...I think using Stellar Assets is the preferred approach as, it is our own library with a nice API. If we keep the wrapper, we lose out on all of the intuitive and code-completion benefits that assets provides.